src/leapr.hpp

```diff
@@ -8,11 +8,25 @@
 #include <range/v3/all.hpp>
 #include <variant>
 #include "endout/endout.h"
+#include <string>
 using namespace njoy;
 #include "lipservice.hpp"
 
+namespace njoy {
+namespace LEAPR {
+
+class LEAPR{
+
+public:
+void operator()( const nlohmann::json& jsonInput,
+                 std::ostream& output,
+                 std::ostream& error,
+                 const nlohmann::json& ){
+
+  error << "this is an example of an error" << std::endl;
```

Comment:
I'm guessing this was just a trial comment and that it no longer needs to be here.